_source/logzio_collections/_log-sources/filebeat-kubernetes.md

```diff
@@ -72,7 +72,10 @@ Run the following command to download the file:
 wget https://raw.githubusercontent.com/logzio/logz-docs/master/shipping-config-samples/filebeat-standard-configuration.yaml
 ```
 
-Apply your custom configuration to the paramaters under `filebeat.yml` and only there. The filebeat.yml field contains a basic Filebeat configuration. You should not change the 'fields' and 'output' fields (indicated in the example below). See Elastic documentation to [learn more about Filebeat configuration options](https://www.elastic.co/guide/en/beats/filebeat/current/configuring-howto-filebeat.html).
```

Comment:
@shalper @mirii1994, one last change.
we don't want to restrict our customers, I have removed the constraints on 'fields'.and added a note regarding the token.